src/ui/AppBar.tsx

```diff
@@ -0,0 +1,11 @@
+import { TitlePortal, AppBar as RaAppBar } from "react-admin";
+import { Box } from "@mui/material";
+export const AppBar = () => {
+  return (
+    <RaAppBar color="transparent">
+      <TitlePortal />
+      <h4>RA E-commerce</h4>
```

Comment:
That's weird: you'll get this *in addition* to the page title. Is it what you want?

Reply:
It was to replace the logo. But I can add a logo like in the demo